serving/docs/lmi/user_guides/lmi-dist_user_guide.md

```diff
@@ -79,6 +79,14 @@ OPTION_MAX_ROLLING_BATCH_SIZE=64
 
 You can follow [this example](../deployment_guide/deploying-your-endpoint.md#configuration---environment-variables) to deploy a model with environment variable configuration on SageMaker.
 
+### LoRA Adapter Support
+
+vLLM has support for LoRA adapters using the [adapters API](../../adapters.md).
+In order to use the adapters, you must begin by enabling them by setting `option.enable_lora=true`.
+Following that, you can configure the LoRA support through the additional settings `option.max_loras`, `option.max_lora_rank`, `option.lora_extra_vocab_size`, and `option.max_cpu_loras`.
```

Comment:
where could I find the doc to see how to use these args?

Reply:
It is on the same page (just scroll down a bit)